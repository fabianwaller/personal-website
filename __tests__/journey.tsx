import About from "@/app/about/page";
import Journey from "@/app/journey/page";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("Journey", () => {
  it("renders journey page unchanged", () => {
    const component = render(<Journey />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
