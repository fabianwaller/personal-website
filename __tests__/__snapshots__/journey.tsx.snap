// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Journey renders journey page unchanged 1`] = `
<DocumentFragment>
  <section
    class="w-full py-8"
    id="journey"
  >
    <div
      class="container relative ml-auto mr-auto w-screen max-w-custom px-8 md:px-0 mb-12"
    >
      <div
        class="text-center"
      >
        <h2>
          My journey
        </h2>
        <span
          class="block text-sm"
        >
          resume
        </span>
      </div>
    </div>
    <div
      class="container relative ml-auto mr-auto w-screen max-w-custom px-8 md:px-0"
    >
      <div
        class="flex flex-col items-center justify-center gap-4"
      >
        <div
          class="w-[500px] max-w-full"
          data-orientation="horizontal"
          dir="ltr"
        >
          <div
            aria-orientation="horizontal"
            class="h-10 items-center justify-center rounded-md p-1 text-muted-foreground mb-6 grid w-full grid-cols-2 bg-background"
            data-orientation="horizontal"
            role="tablist"
            style="outline: none;"
            tabindex="0"
          >
            <button
              aria-controls="radix-:r0:-content-education"
              aria-selected="true"
              class="inline-flex items-center justify-center whitespace-nowrap rounded-sm px-3 py-1.5 text-sm font-semibold ring-offset-background transition-all hover:text-primary-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 data-[state=active]:bg-background data-[state=active]:text-primary"
              data-orientation="horizontal"
              data-radix-collection-item=""
              data-state="active"
              id="radix-:r0:-trigger-education"
              role="tab"
              tabindex="-1"
              type="button"
            >
              Education
            </button>
            <button
              aria-controls="radix-:r0:-content-experience"
              aria-selected="false"
              class="inline-flex items-center justify-center whitespace-nowrap rounded-sm px-3 py-1.5 text-sm font-semibold ring-offset-background transition-all hover:text-primary-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 data-[state=active]:bg-background data-[state=active]:text-primary"
              data-orientation="horizontal"
              data-radix-collection-item=""
              data-state="inactive"
              id="radix-:r0:-trigger-experience"
              role="tab"
              tabindex="-1"
              type="button"
            >
              Experience
            </button>
          </div>
          <div
            aria-labelledby="radix-:r0:-trigger-education"
            class="mt-2 ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2"
            data-orientation="horizontal"
            data-state="active"
            id="radix-:r0:-content-education"
            role="tabpanel"
            style="animation-duration: 0s;"
            tabindex="0"
          >
            <div
              class="grid grid-cols-[max-content_1fr] gap-x-4 sm:gap-x-6"
            >
              <div
                class="sm:px-2"
              >
                <span
                  class="block h-[13px] w-[13px] rounded-full bg-primary"
                />
                <span
                  class="block w-[1px] bg-primary-foreground"
                  style="height: calc(100% - .5rem - 13px); transform: translate(6px, .25rem);"
                />
              </div>
              <div
                style="opacity: 0; transform: none;"
              >
                <div
                  class="border border-border shadow-custom hover:bg-card-foreground grid gap-4 overflow-hidden rounded-lg bg-card px-6 py-4 mb-6"
                >
                  <div
                    class=""
                  >
                    <div
                      class="flex flex-col justify-center gap-2 items-start"
                    >
                      <div>
                        <h3
                          class="text-lg font-medium leading-none tracking-tight text-title-normal"
                        >
                          Bachelor of Computer Science
                        </h3>
                        <p
                          class="text-normal"
                        >
                          University of Saarland, Saarbr√ºcken, Germany
                        </p>
                      </div>
                      <p>
                        2021 - 2024
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="grid grid-cols-[max-content_1fr] gap-x-4 sm:gap-x-6"
            >
              <div
                class="sm:px-2"
              >
                <span
                  class="block h-[13px] w-[13px] rounded-full bg-primary"
                />
              </div>
              <div
                style="opacity: 0; transform: none;"
              >
                <div
                  class="border border-border shadow-custom hover:bg-card-foreground grid gap-4 overflow-hidden rounded-lg bg-card px-6 py-4 mb-6"
                >
                  <div
                    class=""
                  >
                    <div
                      class="flex flex-col justify-center gap-2 items-start"
                    >
                      <div>
                        <h3
                          class="text-lg font-medium leading-none tracking-tight text-title-normal"
                        >
                          Abitur
                        </h3>
                        <p
                          class="text-normal"
                        >
                          Peter-Wust-Gymnasium
                        </p>
                      </div>
                      <p>
                        2013 - 2021
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-labelledby="radix-:r0:-trigger-experience"
            class="mt-2 ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2"
            data-orientation="horizontal"
            data-state="inactive"
            hidden=""
            id="radix-:r0:-content-experience"
            role="tabpanel"
            tabindex="0"
          />
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
